paparazzi/build.gradle

```diff
@@ -1,10 +1,10 @@
-import app.cash.paparazzi.gradle.UnpackAarKt
 import org.jetbrains.kotlin.gradle.plugin.AbstractKotlinPluginKt
 
 apply plugin: 'org.jetbrains.kotlin.jvm'
 apply plugin: 'org.jetbrains.dokka'
 apply plugin: 'com.vanniktech.maven.publish'
 apply plugin: 'dev.drewhamilton.poko'
+apply from: "customConfiguration.gradle"
```

Comment:
looks like gradle file translation from kotlin is not working as expected. Causes issue with downstream IR kotlin compilation.

Reply:
required me to add `implementation libs.plugin.kotlin` to the buildSrc/build.gradle file to get compilation working.